src/encoder.rs

```diff
@@ -305,6 +384,93 @@ impl<W: Write> Writer<W> {
     pub fn into_stream_writer_with_size(self, size: usize) -> StreamWriter<'static, W> {
         StreamWriter::new(ChunkOutput::Owned(self), size)
     }
+
+    pub fn write_separate_default_image(&mut self, data: &[u8]) -> Result<()> {
```

Comment:
This seems odd. When transitioning something to animated PNG I would expect the opposite, that I need one or two more calls to setup the animation. I would however not expect that the still image, as which this chunk is interpreted when it is written, needs a separate methods. Consider two scenarios: 
* The image is generated by a separate method whose parameter is `Writer<W>`.
* An image is modified in a pipeline.

In both cases it's complicated that the reader has one method for all images but the `write_image` is only for animated images if the image is animated.

Reply:
(You meant `write_separate_default_image` at the end right?)

This could be solved by moving the method to the `Encoder` creation, with like a `set_separate_default_image(data: &[u8])` that will panic/error if the animation parameters are not set (or sets the default parameters for an animation)

Reply:
Another would be to have a mandatory additional step where the `FrameControl` data needs to be initialized, independent of the call that initalized the animation control. Then treat every call to `write_image_data` as writing the default image as long as no frame control was set. That's very close to the way the chunks are organized in the image and avoids complicating the constructor.